README.md

```diff
@@ -211,13 +211,17 @@ When a new version of allms is ready to be released, do the following operations
     2. `git push origin <NEW-VERSION>`
 
 3. **Publish package to PyPI**:
-    1. Go to _Actions_ → _Manual Publish To PyPI_;
-    2. Select "master" as branch and click _Run workflow_;
+    1. Go to _Actions_ → _Release_;
```

Comment:
Is publishing to PyPI needed before github release? Otherwise one can create tag in the release and skip adding the tag manually

Reply:
> Is publishing to PyPI needed before github release? 

No, it's not needed :D 

> Otherwise one can create tag in the release and skip adding the tag manually

That's a nice idea. You can update the README and implement this change.